### PR TITLE
CTDA-1916

### PR DESCRIPTION
**Pull request by RobertLaverick**
fixed guarantee-concatination bug in EnsureDefaultGuaranteeService

### CODE REVIEW

**Review by derekspalmer — APPROVED** [review]

**Review by drnaylor-oc — APPROVED** [review]

**Comment by RobertLaverick** [comment]
Waiting on a confirmation from Scott and Oli before I merge this, but it's good to see it's approved.

**Comment by RobertLaverick** [comment]
Merging after action decided by leads to proceed with introducing the fix.

**Pull request merged**
